crates/bevy_pbr/src/pbr_material.rs

```diff
@@ -1074,7 +1075,10 @@ impl AsBindGroupShaderType<StandardMaterialUniform> for StandardMaterial {
             clearcoat: self.clearcoat,
             clearcoat_perceptual_roughness: self.clearcoat_perceptual_roughness,
             anisotropy_strength: self.anisotropy_strength,
-            anisotropy_rotation,
+            anisotropy_rotation: vec2(
+                self.anisotropy_rotation.cos(),
+                self.anisotropy_rotation.sin(),
+            ),
```

Comment:
Why move this here?

Reply:
Sorry, had originally started wrapping cfg around all the values and this fixed a dead code warning, then noticed that wasn’t the case with clearcoat so I did a bit but accidentally left this. Should make no difference?